src/Aspire.Hosting.AWS/CDK/Utils/ConstructExtensions.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Amazon.CDK;
+using Constructs;
+
+namespace Aspire.Hosting.AWS.CDK;
+
+internal static class ConstructExtensions
+{
+    public static string StackUniqueId(this IConstruct construct)
```

Comment:
Some docs to help maintainers understand what this is does and why it is needed would be really helpful.